reactivesocket-tck-drivers/build.gradle

```diff
@@ -19,10 +18,18 @@ dependencies {
     compile project(':reactivesocket-core')
     compile project(':reactivesocket-client')
     compile project(':reactivesocket-transport-tcp')
+    testCompile project(':reactivesocket-test')
     compile 'com.fasterxml.jackson.core:jackson-core:2.8.0.rc2'
     compile 'com.fasterxml.jackson.core:jackson-databind:2.8.0.rc2'
     compile 'org.apache.commons:commons-lang3:3.4'
     compile 'io.reactivex:rxjava-reactive-streams:1.1.0"'
     compile 'io.airlift:airline:0.7'
 }
 
+task (runTests, dependsOn: jar, type: JavaExec) {
+    classpath(sourceSets.main.runtimeClasspath, sourceSets.main.compileClasspath)
+    main = 'io.reactivesocket.tckdrivers.main.TestMain'
+    args '--port',  '4567', '--serverfile',  'src/test/resources/server$.txt',  '--clientfile',  'src/test/resources/client$.txt'
+}
+
+build.finalizedBy(runTests)
```

Comment:
I don't think this is an appropriate use of finalizedBy, it looks like its meant for cleanup e.g. stopping a SQL db you launched and used multiple times in a test run.

e.g.

http://buransky.com/programming/integration-testing-with-gradle/
http://selimober.com/blog/2014/01/24/separate-unit-and-integration-tests-using-gradle/